### PR TITLE
Change plugin manager styles

### PR DESCRIPTION
**Pull request by alexcherman**

### CODE REVIEW

File: src/app/components/plugin-manager-component.js [hunk 1]
```diff
@@ -37,12 +47,15 @@ const css = csjs`
...
-    padding: 0 7px;
-    font-weight: bolder;
-    margin-top: 5px;
-    text-transform: lowercase;
+    padding: 4px;
+    margin: 0 8px;
+    font-weight: 700;
+    font-size: 9px;
+    line-height: 12px;
+    text-transform: uppercase;
```
Comment (ryestew) [inline src/app/components/plugin-manager-component.js]: Would this be better as a badge? - I realize that it is currently coded with .versionWarning.

File: src/app/components/side-panel.js [hunk 2]
```diff
@@ -13,7 +13,6 @@ const css = csjs`
   }
   .swapitTitle {
     margin: 0;
-    text-transform: uppercase;
```
Comment (ryestew) [inline src/app/components/side-panel.js]: Lets discuss with Natalia about this.

Comment (alexcherman) [reply src/app/components/side-panel.js]: Natalia doesn't mind if there are uppercase letters so I made uppercased it again

File: src/app/components/plugin-manager-component.js (hunk 1 above)
Comment (alexcherman) [reply src/app/components/plugin-manager-component.js]: Made it as .badge-light. Check it please

Comment (ryestew) [reply src/app/components/plugin-manager-component.js]: Sorry - I don't see any .badge-light on plugin-manager-component.js
Please point me to where you changed it.  

Also I think there will be some design adjustment to this page - I'll be reviewing that with Natalia on Thursday.

File: src/app/components/side-panel.js [hunk 3]
```diff
@@ -122,11 +122,11 @@ export class SidePanel extends AbstractPanel {
       name = profile.displayName ? profile.displayName : profile.name
       docLink = profile.documentation ? yo`<a href="${profile.documentation}" class="${css.titleInfo}" title="link to documentation" target="_blank"><i aria-hidden="true" class="fas fa-book"></i></a>` : ''
       if (profile.version && profile.version.match(/\b(\w*alpha\w*)\b/g)) {
-        versionWarning = yo`<small title="Version Alpha" class="${css.versionWarning}">alpha</small>`
+        versionWarning = yo`<small title="Version Alpha" class="badge-light ${css.versionBadge}">alpha</small>`
       }
```
Comment (alexcherman) [inline src/app/components/side-panel.js]: @ryestew I added badge here

File: src/app/components/side-panel.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ const css = csjs`
   .titleInfo {
     padding-left: 10px;
   }
-  .versionWarning {
+  .versionBadge {
     background-color: var(--light);
```
Comment (alexcherman) [inline src/app/components/side-panel.js]: @ryestew And changed class from .versionWarning to .versionBadge

File: src/app/components/plugin-manager-component.js [hunk 5]
```diff
@@ -104,16 +119,16 @@ class PluginManagerComponent extends ViewPlugin {
...
-        <div class="${css.row} justify-content-between align-items-center">
-          <h6 class="${css.displayName}">
-            ${displayName}
-            ${versionWarning}
-          </h6>
-          ${activationButton}
-        </div>
-        <p class="${css.description}">${api.profile.description}</p>
-      </article>
+    <article id="remixPluginManagerListItem_${name}" class="list-group-item px-2 pt-2 pb-0 plugins-list-group-item" title="${displayName}" >
```
Comment (LianaHus) [inline src/app/components/plugin-manager-component.js]: we need an indent here. a tab size

Comment (alexcherman) [reply src/app/components/plugin-manager-component.js]: @LianaHus Click view file please

Comment (alexcherman) [reply src/app/components/plugin-manager-component.js]: Here you see narrowed version

Comment (LianaHus) [reply src/app/components/plugin-manager-component.js]: sorry but we need a space.
```
return yo`
 blablaba
 blablabla
`
```

Comment (LianaHus) [reply src/app/components/plugin-manager-component.js]: ehat you have is
```
return yo`
blablaba
blablabla
`
```

**Comment by LianaHus** [comment]
I see some problems with position of "settings" button and "active/inactive modul" (why not plugins btw?) in different themes. here are screenshots

q1
q2
q3


z1
z2
z3

**Comment by LianaHus** [comment]
also why the footer has opacity in dark theme? looks weird and broken to me. 
@ryestew

**Comment by alexcherman** [comment]
> also why the footer has opacity in dark theme? looks weird and broken to me.
> @ryestew

It's not broken. I made according to design.

**Comment by alexcherman** [comment]
> I see some problems with position of "settings" button and "active/inactive modul" (why not plugins btw?) in different themes. here are screenshots
> 
> q1
> q2
> q3
> 
> z1
> z2
> z3

Ok, I see. It should be on the same line. Will fix it

**Comment by alexcherman** [comment]
Thanks for the screenshots.

**Comment by alexcherman** [comment]
@LianaHus By "active/inactive module" position problem you mean spaces from left and right sides, don't you? I made those spaces also according to design

**Comment by LianaHus** [comment]
about opacity I know, this is why I tagged Rob too. But Why we have opacity only on dark theme?

**Comment by LianaHus** [comment]
yes, the spaces. I don't mind to have spaces for active/inactive modules" but it is different in different themes?

File: src/app/components/plugin-manager-component.js [hunk 6]
```diff
@@ -164,36 +179,36 @@ class PluginManagerComponent extends ViewPlugin {
...
+        <span class="h6 mb-0 mr-2">Active Modules</span>
+        <span class="badge badge-pill badge-warning">${actives.length}</span>
       </nav>`
       : ''
     const inactiveTile = inactives.length !== 0
       ? yo`
-      <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between align-items-center">
-        <span class="navbar-brand">Inactive Modules</span>
-        <span class="badge badge-pill badge-primary" style = "cursor: default;">${inactives.length}</span>
+      <nav class="plugins-list-header justify-content-start navbar navbar-expand-lg bg-light navbar-light align-items-center px-4 py-3">
```
Comment (LianaHus) [inline src/app/components/plugin-manager-component.js]: px-4 you added here is braking other themes, can't you fix this in nav classes definitions in a theme directly?

File: src/app/components/plugin-manager-settings.js [hunk 7]
```diff
@@ -126,8 +130,8 @@ export class PluginManagerSettings {
 
   render () {
     return yo`
-    <footer class="navbar navbar-light bg-light ${css.permissions}">
-      <button onclick="${() => this.openDialog()}" class="btn btn-info">Settings</button>
+    <footer class="bg-light ${css.permissions} remix-bg-opacity">
```
Comment (ryestew) [inline src/app/components/plugin-manager-settings.js]: I think this definition of remix-bg-opacity should be defined in the csjs of this file and not in the bootstrap theme.  Let me know if you think there is an advantage to leaving it in the theme's css.

**Comment by ryestew** [comment]
The PR is nearly there and we'd like to merge it soon- these issues are small - so just to summarize:
- the indent on line 122 of src/app/components/plugin-manager-component.js 
- for the footer of the plugin manager: I think the definition of the class remix-bg-opacity should be defined in the csjs and not in the theme's css. 
- the active/inactive module badge - should be the same color (so not badge-alert) & shape ( so remove badge-pill) as the figma
- the active /inactive modules headline ( the h6) are now aligned with the text below - which is correct - but this has moved these headlines in the other themes - ideally you'd make it so that the change only happens in the EthWorks theme.

I've put these issues in the trello board too.

**Comment by alexcherman** [comment]
I

> The PR is nearly there and we'd like to merge it soon- these issues are small - so just to summarize:
> 
> * the indent on line 122 of src/app/components/plugin-manager-component.js
> * for the footer of the plugin manager: I think the definition of the class remix-bg-opacity should be defined in the csjs and not in the theme's css.
> * the active/inactive module badge - should be the same color (so not badge-alert) & shape ( so remove badge-pill) as the figma
> * the active /inactive modules headline ( the h6) are now aligned with the text below - which is correct - but this has moved these headlines in the other themes - ideally you'd make it so that the change only happens in the EthWorks theme.
> 
> I've put these issues in the trello board too.

I think we should leave remix-bg-opacity class because this change should only happen in Ethworks theme

**Comment by LianaHus** [comment]
thanks for updates!

The last thing is the settings button is still shifted in old themes
> > q2

**Comment by alexcherman** [comment]
> thanks for updates!
> 
> The last thing is the settings button is still shifted in old themes
> 
> > > q2

@LianaHus  Done.

**Comment by LianaHus** [comment]
> > thanks for updates!
> > The last thing is the settings button is still shifted in old themes
> > > > q2
> 
> @LianaHus Done.

Cool! When do we merge @ryestew ?

**Pull request merged**